app/services/bulk_upload/processor.rb

```diff
@@ -6,12 +6,13 @@ def initialize(bulk_upload:)
   end
 
   def call
+    bulk_upload.update!(processing: true)
```

Comment:
I think we either want to set processing to true as soon as we create the bulk upload, or we want another status, something like `scheduled` etc., because depending on how available the queue is it won't immediately start processing the file, and we'd get `Logs uploaded with no errors` status for those bulk uploads. 

You can check this if you try uploading multiple files right after each other

Reply:
Ah okay, that's a good idea, I was unaware of the queue, will set it to true when the bulk upload is created.